### PR TITLE
OPENDNSSEC-721 SoftHSMv2

### PR DESCRIPTION
**Pull request by halderen**
- drop unused functions searching for C++ compiler for running tests
- improve the parameter substitution for running tests

### CODE REVIEW

**Comment by halderen** [comment]
#build

**Comment by halderen** [comment]
Can one of the admins verify this patch?

**Comment by halderen** [comment]
#build

File: testing/make.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+if [ \! -f $INSTALL_ROOT/.softhsm2.ok ] ; then
+  rm -f softhsm-2.2.0.tar.gz
+  wget 'https://dist.opendnssec.org/source/softhsm-2.2.0.tar.gz'
+  gzip -d < softhsm-2.2.0.tar.gz | tar xf -
+  cd softhsm-2.2.0
+  ./configure --prefix="$INSTALL_ROOT" \
+              --disable-non-paged-memory \
+              --with-migrate --disable-p11-kit \
+              --with-crypto-backend=openssl \
+              --disable-ecc --disable-gost
```
Comment (yschaeff) [inline testing/make.sh]: What does it take to do ECC/GOST as well? - creating an ECC test is still on my TODO list.

**Comment by yschaeff** [comment]
ACK.

File: testing/make.sh (hunk 1 above)
Comment (halderen) [inline testing/make.sh]: Not all test machines are/were equiped with the right openssl lib afaik/earlier tested.  This is just part of the default distribution.
For my part, not part of this issue.

**Pull request merged**